packages/edge-login-ui-rn/src/components/themed/BackButton.js

```diff
@@ -12,36 +12,61 @@ import {
   withTheme
 } from '../services/ThemeContext'
 
+const isIos = Platform.OS === 'ios'
+
 type Props = {
   onPress(): void,
-  marginRem?: number[] | number
+  marginRem?: number[] | number,
+  enabled?: boolean
 }
 
 const BackButtonComponent = ({
   theme,
   marginRem,
-  onPress
+  onPress,
+  enabled = true
```

Comment:
If we only use `<BackButton ... enabled={false}/>` to change the `enabled` flag, I think it would be better to turn it over to `<BackButton ... disabled/>` and `disabled = false`